public/examples/upload/uploadApp.js

```diff
@@ -345,7 +345,11 @@
         }
         var baseUrl = (version === 'IPv6') ? 'http://' + testPlan.baseUrlIPv6 : 'http://' + testPlan.baseUrlIPv4;
 
-        var uploadHttpConcurrentTestSuite = new window.uploadHttpConcurrentProgress(baseUrl + '/upload', 'POST', 1, 15000, 15000, 2, uploadHttpOnComplete, uploadHttpOnProgress,
+        var concurrentRuns = 1;
+        if(uploadSize>= 10671896){
```

Comment:
Can we have `10671896` as a variable/const/config value? With a comment on what this is tracking. The same in the speed-testJS.js